honeybee_vtk/file_writers.py

```diff
@@ -0,0 +1,229 @@
+
+    # write grid points to a vtk file if grids are found in HBJSON
+    grids = check_grid(hbjson)
+
+    # If grids are there in HBJSON and they are requested
+    if grids and include_grids:
+        grid_file_names = _write_grids(grids, vtk_writer, vtk_extension, target_folder)
+        file_names.extend(grid_file_names)
+
+    # Write vectors if they are requested
+    if include_vectors:
+        to_write_vectors = [hb_types, grouped_points, grids, include_grids, vtk_writer,
+                            vtk_extension, target_folder]
+        vector_file_names = _write_vectors(*to_write_vectors)
+        file_names.extend(vector_file_names)
+
+    # Write index.json
+    write_index_json(target_folder, file_names)
+
+    # Collect everything in a .zip
+    # Use the name of HBJSON if file name is not provided
+    if not file_name:
+        name = '.'.join(os.path.basename(file_path).split('.')[:-1])
+        file_name = clean_string(name)
+
+    # remove extension if provided by user
+    file_name = file_name if not file_name.lower().endswith('.zip') else file_name[:-4]
+
+    # Create a .zip file to capture all the generated .vtk files
+    zip_name = file_name
+    file_path = os.path.join(target_folder, file_name)
+    shutil.make_archive(file_path, 'zip', target_folder)
+ 
+    # Delete the index.json if there is permission
+    try:
+        file_name = os.path.join(target_folder, 'index.json')
+        os.remove(file_name)
+    except OSError:
+        warnings.warn(
+            'Honeybee does not have permission to delete index.json.'
+            ' You may please delete the .vtk files manually.'
+        )
+    # Delete the folders containing json datasets if there is permission
+    for file_name in file_names:
+        try:
+            file_name = os.path.join(target_folder, file_name)
+            shutil.rmtree(file_name)
+        except OSError:
+            warnings.warn(
+                f'Honeybee does not have permission to delete {file_names}.'
+                ' You may please delete the .vtk files manually.'
+            )
+    
+    # Setting file names for three different formats
+    zip_file = os.path.join(target_folder, zip_name + '.zip')
+    vtkjs_file = os.path.join(target_folder, zip_name + '.vtkjs')
+    html_file = os.path.join(target_folder, zip_name + '.html')
+
+    # Renaming .zip to .vtkjs
+    os.rename(zip_file, vtkjs_file)
+
+    # Path to the Paraview Glance template
+    html_path = '../honeybee-vtk/Paraview Glance/ParaViewGlance.html'
+
+    # Embed json datasets in HTML
+    if os.path.exists(html_path):
+        convertDirectoryToZipFile(vtkjs_file)
+        addDataToViewer(vtkjs_file, html_path)
+    
+    # Delete vtkjs file
+    try:
+        os.remove(vtkjs_file)
+    except OSError:
+        warnings.warn(
+            'Honeybee does not have permission to delete index.json.'
+            ' You may please delete the .vtk files manually.'
+        )
+    
+    webbrowser.open(html_file)
```

Comment:
This should be an option. What if I just want to create the HTML file in a recipe and don't open it up?